frontend/packages/knative-plugin/src/components/add/event-sources/YAMLEditorSection.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import YAMLEditorField from '@console/shared/src/components/formik-fields/YAMLEditorField';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { useField } from 'formik';
+
+const YAMLEditorSection: React.FC = () => {
+  const [field] = useField('type');
+  return (
+    <FormSection fullWidth title={field.value}>
+      <YAMLEditorField name="yamlData" />
```

Comment:
it will be good have `onSave` here as well to validate yaml as and when it gets Saved

```
<YAMLEditorField name="yamlData" onSave={handleSubmit}  />
```

Reply:
FYI, this `onSave` is triggered when `CTRL+S` or `CMD+S` is pressed by the user. If we don't want that key binding to submit the form in case there are still some form fields left then we should not add this.